### DISCUSSION TITLE
Initial Prompt?

### DISCUSSION THREAD

**Discussion by sailingbo** [KoljaB/RealtimeSTT#41] [Q&A] [answered]
Is it possible to pass "initial_prompt" and "suppress_tokens" values to be considered by the model when run?

**Comment by KoljaB** [comment, marked as answer]
Currently not, no prob to add tho. Do you need different parameters for main and realtime transcription models or do you think one parameter for init prompt and suppress tokens is sufficient?

**Comment by sailingbo** [comment]
Amazing. I'm focused on realtime, so that's the only one I care about.

**Reply by sailingbo** [reply]
And I'd be curious to know if there's another way to add a list of vocabulary keywords that the model will weigh higher than other words.

**Comment by KoljaB** [comment]
There is only [grammar sampling](https://twitter.com/ggerganov/status/1735334112230097360) as of my knowledge and this restricts to certain words.
